src/renderer/atlas/AtlasEngine.api.cpp

```diff
@@ -673,7 +675,7 @@ void AtlasEngine::_resolveFontMetrics(const wchar_t* requestedFaceName, const Fo
             // Since clients expect that settings the font height to Y yields back a font height of Y,
             // we're scaling the X relative/proportional to the actual cellWidth/cellHeight ratio.
             // The code below uses a poor form of integer rounding.

```

Comment:
This comment may no longer be true ;) (is it?)

Reply:
Ah thanks for pointing this out. The code didn't work correctly anymore. Luckily this particular code remains entirely unused in Windows Terminal (it's only relevant to console APIs).